lib/docker.ts

```diff
@@ -96,6 +107,50 @@ class Docker {
   }
 
   public async save(targetImage: string, destination: string) {
+    const request = {
+      path: `/images/${targetImage}/get?`,
+      method: "GET",
+      isStream: true,
+      statusCodes: {
+        200: true,
```

Comment:
why do we need these status codes? how does Modem react to them?

Reply:
Status codes are utilised here: https://github.com/apocas/docker-modem/blob/master/lib/modem.js#L299. Basically if the code isn't true it will return an error that incorporates the string provided.